components/Layout/Layout.tsx

```diff
@@ -1,6 +1,7 @@
 import Head from 'next/head'
 import styles from './Layout.module.scss'
 import Header from '../Header/Header'
+import Footer from '../Footer/Footer'
```

Comment:
That imports work fine but I provided something what is called `absolute imports` so you don't have to use this dots `../../../..`. It is useful because you don't need to find directories by path but only you can use sth like a `import Layout from "@/components/Layout/Layout";` and you can import any component from components directory from any place. Check how it is done in other files like history.tsx